boardswarm/README.md

```diff
@@ -1,9 +1,9 @@
 # Boardswarm server
 
 Boardswarm is a gRPC server exposing APIs useful for interacting with
-development boards. The swarm part of the names comes from the ability of a
+development boards. The "swarm" part of the name comes from the ability of a
 server to connect to other boardswarm instances, sharing and distributing
-functionality transparantly for the end-user.
+functionality transparently for the end-user.
```

Comment:
@obbardc fwiw if you do suggestions it would be cool if you can add them as github suggestions so it's a simple click to fold it in :). Fwiw this typo fix is already a step forward so i'd be fine with either version